controllers/user.js

```diff
@@ -39,9 +43,23 @@ const getCurrentUser = async (userId) => {
   return user;
 };
 
+const makeAvatar = async (userId, filePath) => {
```

Comment:
1. **Rotacja obrazu**:
   - Usuń `.rotate(360)`, ponieważ nie zmienia obrazu i może być zbędne?

2. **Asynchroniczne usuwanie plików**:
   - Możesz zmienić `fs.unlinkSync` na `fs.unlink` aby nie blokować pętli zdarzeń Node.js.

3. **Bezpieczeństwo i walidacja**:
   - Upewnij się, że `userId` i `filePath` są prawidłowo sprawdzane przed użyciem.

4. **Zwracane wartości funkcji**:
   - Rozważ, czy funkcja powinna zwracać pełny URL lub status operacji zamiast samej nazwy pliku.

5. **Dokumentacja i komentarze**:
   - Dodaj komentarze wyjaśniające działanie funkcji, co ułatwi jej późniejsze utrzymanie.

Reply:
1) rotate skasowany - z wykładów mi się coś ubzdurało że to może sprawdzać czy plik jest faktycznie obrazem bo innych się nie da od tak obracać xd
2) unlink poprawiony i dopisane promises do fs na początku
3) user id jest pobierane z zachaszowanego taga, a filepath ma teraz walidację
4) zostawiłem tak jak jest ale dodałem do zwrotki /avatars/+nazwa co zmieniło zwrotną wiadomość w klikalny link do nowego avatara
5) zdecydowanie muszę dodać więcej komentarzy bo się gubię we własnym kodzie